app/controllers/posts_controller.rb

```diff
@@ -29,4 +29,13 @@ def create
       end
     end
   end
+
```

Comment:
- [ ] Please use **CanCanCan** for Post authorization in `Post Controller`:
- `load_and_authorize_resource` method is provided to automatically authorize all actions in a RESTful style resource controller. Here is the code: 
**class PostsController < ApplicationController
  load_and_authorize_resource**
- Open this Link [cancancan](https://github.com/CanCanCommunity/cancancan) and scroll down to Controller helper section, to Learn more